### PR TITLE
Fixed layout and styles for the branch-contacts-info block on the branche page

### PR DESCRIPTION
**Pull request by aleevas**
This commit should fix broken layout of branch info block on the branche page.
Currently it looks like:
CR-57-oldView

After fix it looks like that:
CR-57-newView

## Steps for review

- [x] Create or visit a branche page.
- [x] If you need to create it, please fill all fields in `Contact Info` section.
- [x] Then in the` Header Area.` please place the `Branch Contacts Info` block
- [x] Save node and check block on the page.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss [hunk 1]
```diff
@@ -0,0 +1,200 @@
+.paragraph--type--branch-contacts-info {
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss:1]: I can't find any css changes in the PR
I believe there is a step missing - need to recompile css

**Review by podarok — CHANGES_REQUESTED** [review]

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss (hunk 1 above)
Comment (podarok) [reply themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss:1]: @aleevas

Comment (aleevas) [reply themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss:1]: @podarok thank you for review.
Could you help me to understand that I need to do more?
Should I just make a recompile style and add that changes (css files) to my PR?

Comment (aleevas) [reply themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss:1]: @podarok
I've re-compiled styles.
Is it what I should did?
Please check it again.

**Comment by podarok** [comment]
ok to test

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss (hunk 1 above)
Comment (podarok) [reply themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss:1]: Yes, this is great now, thank you @aleevas

**Review by podarok — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2683/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2683/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2683/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2683/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2683/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2683/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2683/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2683/

**Pull request merged**